### ISSUE TITLE
Problem running train_mesh_3dpw

### ISSUE THREAD

**Issue by UsanoCoCr** [TangTao-PKU/ARTS#2]
While running the script train_mesh_3dpw, I noticed that the validation results in the log got progressively higher:

Epoch1 (500/515) => surface error: 84.0549, joint error: 66.6006: 100%|█████████████████████████████████████| 515/515 [04:57<00:00,  1.73it/s]
Epoch1 MPVPE: 87.77, MPJPE: 72.74
Current epoch 2, lr: 3e-05
Epoch2_(20000/20002) => vl: 0.018 nl: 0.021 el: 0.000 m->j: 0.015 evl: 0.150 ljl: 0.051 il: 0.079 sl: 0.006 pal: 0.013 tl: 0.039: 100%|█| 2000
Epoch2 Loss: 0.0339
Epoch2 (500/515) => surface error: 85.7004, joint error: 67.6811: 100%|█████████████████████████████████████| 515/515 [05:08<00:00,  1.67it/s]
Epoch2 MPVPE: 88.41, MPJPE: 73.87
Current epoch 3, lr: 2.7000000000000002e-05
Epoch3_(20000/20002) => vl: 0.012 nl: 0.029 el: 0.024 m->j: 0.010 evl: 0.132 ljl: 0.059 il: 0.059 sl: 0.005 pal: 0.013 tl: 0.052: 100%|█| 2000
Epoch3 Loss: 0.0580
Epoch3 (500/515) => surface error: 85.7742, joint error: 63.3131: 100%|█████████████████████████████████████| 515/515 [05:06<00:00,  1.68it/s]
Epoch3 MPVPE: 89.37, MPJPE: 73.41
Current epoch 4, lr: 2.7000000000000002e-05
Epoch4_(20000/20002) => vl: 0.017 nl: 0.023 el: 0.018 m->j: 0.014 evl: 0.164 ljl: 0.086 il: 0.052 sl: 0.003 pal: 0.011 tl: 0.051: 100%|█| 2000
Epoch4 Loss: 0.0524
Epoch4 (500/515) => surface error: 78.1445, joint error: 60.3662: 100%|█████████████████████████████████████| 515/515 [05:06<00:00,  1.68it/s]
Epoch4 MPVPE: 90.13, MPJPE: 74.24
Current epoch 5, lr: 2.43e-05
Epoch5_(20000/20002) => vl: 0.018 nl: 0.018 el: 0.015 m->j: 0.014 evl: 0.189 ljl: 0.121 il: 0.055 sl: 0.002 pal: 0.012 tl: 0.049: 100%|█| 2000
Epoch5 Loss: 0.0487
Epoch5 (500/515) => surface error: 82.3930, joint error: 64.5368: 100%|█████████████████████████████████████| 515/515 [05:09<00:00,  1.66it/s]
Epoch5 MPVPE: 90.77, MPJPE: 74.83
Current epoch 6, lr: 2.43e-05
Epoch6_(20000/20002) => vl: 0.013 nl: 0.020 el: 0.019 m->j: 0.010 evl: 0.151 ljl: 0.097 il: 0.047 sl: 0.002 pal: 0.011 tl: 0.045: 100%|█| 2000
Epoch6 Loss: 0.0466
Epoch6 (500/515) => surface error: 81.4755, joint error: 62.5663: 100%|█████████████████████████████████████| 515/515 [04:00<00:00,  2.14it/s]
Epoch6 MPVPE: 87.77, MPJPE: 72.68
Current epoch 7, lr: 2.1870000000000002e-05
Epoch7_(20000/20002) => vl: 0.016 nl: 0.017 el: 0.015 m->j: 0.012 evl: 0.196 ljl: 0.129 il: 0.046 sl: 0.001 pal: 0.010 tl: 0.045: 100%|█| 2000
Epoch7 Loss: 0.0443
Epoch7 (500/515) => surface error: 81.6383, joint error: 63.5760: 100%|█████████████████████████████████████| 515/515 [03:58<00:00,  2.16it/s]
Epoch7 MPVPE: 90.16, MPJPE: 74.30
Current epoch 8, lr: 2.1870000000000002e-05
Epoch8_(20000/20002) => vl: 0.012 nl: 0.021 el: 0.014 m->j: 0.010 evl: 0.144 ljl: 0.099 il: 0.040 sl: 0.001 pal: 0.009 tl: 0.038: 100%|█| 2000
Epoch8 Loss: 0.0428
Epoch8 (500/515) => surface error: 79.4619, joint error: 63.4798: 100%|█████████████████████████████████████| 515/515 [03:57<00:00,  2.17it/s]
Epoch8 MPVPE: 89.41, MPJPE: 73.97
Current epoch 9, lr: 1.9683e-05
Epoch9_(20000/20002) => vl: 0.012 nl: 0.014 el: 0.012 m->j: 0.010 evl: 0.170 ljl: 0.120 il: 0.044 sl: 0.001 pal: 0.009 tl: 0.035: 100%|█| 2000
Epoch9 Loss: 0.0411
Epoch9 (500/515) => surface error: 82.7028, joint error: 64.1694: 100%|█████████████████████████████████████| 515/515 [03:58<00:00,  2.16it/s]
Epoch9 MPVPE: 90.28, MPJPE: 74.27

Is this common? I didn't change the configuration. Looking forward to your reply.

**Comment by TangTao-PKU** [comment]
Thank you for your attention to our work！
There was something not quite normal. We retrained the model and found that using the training sets in the train_list: ['PW3D', 'MPII3D', 'Human36M'] can achieve better results, and the training process converges more effectively. We have updated the config file, and at the same time, we have uploaded the pre-trained checkpoints in the readme (https://drive.google.com/drive/folders/1txQWReexVOLjpmgi426vMCw0Itq4K6C0?usp=sharing) to facilitate your testing and the running of the demo.
